### PR TITLE
Currency:Adding build_script,build_info.json for solr

### PR DESCRIPTION
**Pull request by shubham-garud**
## Checklist

- [ ] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [ ] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by shubham-garud** [comment]
Complete!
Cloning into 'solr'...
remote: Enumerating objects: 1260192, done.
remote: Counting objects: 100% (1022/1022), done.
remote: Compressing objects: 100% (549/549), done.
remote: Total 1260192 (delta 418), reused 840 (delta 311), pack-reused 1259170
Receiving objects: 100% (1260192/1260192), 468.44 MiB | 7.56 MiB/s, done.
Resolving deltas: 100% (742475/742475), done.
Updating files: 100% (7590/7590), done.
Note: switching to 'releases/solr/9.2.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 92c5515e291 More smoke test fixes for 9.2
Downloading gradle-wrapper.jar from https://raw.githubusercontent.com/gradle/gradle/v7.6.0/gradle/wrapper/gradle-wrapper.jar
Generating gradle.properties
Downloading https://services.gradle.org/distributions/gradle-7.6-bin.zip
...........10%............20%...........30%............40%............50%...........60%............70%............80%...........90%............100%

Welcome to Gradle 7.6!

Here are the highlights of this release:
- Added support for Java 19.
- Introduced `--rerun` flag for individual task rerun.
- Improved dependency block for test suites to be strongly typed.
- Added a pluggable system for Java toolchains provisioning.

For more details see https://docs.gradle.org/7.6/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)
> Task :buildSrc:compileJava
> Task :buildSrc:compileGroovy NO-SOURCE
> Task :buildSrc:processResources NO-SOURCE
> Task :buildSrc:classes
> Task :buildSrc:jar
> Task :buildSrc:assemble
> Task :buildSrc:compileTestJava NO-SOURCE
> Task :buildSrc:compileTestGroovy NO-SOURCE
> Task :buildSrc:processTestResources NO-SOURCE
> Task :buildSrc:testClasses UP-TO-DATE
> Task :buildSrc:test NO-SOURCE
> Task :buildSrc:check UP-TO-DATE
> Task :buildSrc:build
> Task :localSettings UP-TO-DATE

BUILD SUCCESSFUL in 2m 8s
2 actionable tasks: 2 executed
> Task :buildSrc:compileJava UP-TO-DATE
> Task :buildSrc:compileGroovy NO-SOURCE
> Task :buildSrc:processResources NO-SOURCE
> Task :buildSrc:classes UP-TO-DATE
> Task :buildSrc:jar UP-TO-DATE
> Task :buildSrc:assemble UP-TO-DATE
> Task :buildSrc:compileTestJava NO-SOURCE
> Task :buildSrc:compileTestGroovy NO-SOURCE
> Task :buildSrc:processTestResources NO-SOURCE
> Task :buildSrc:testClasses UP-TO-DATE
> Task :buildSrc:test NO-SOURCE
> Task :buildSrc:check UP-TO-DATE
> Task :buildSrc:build UP-TO-DATE

> Task :randomizationInfo
Running tests with randomization seed: tests.seed=7A6BF4DD83141E98

> Task :errorProneSkipped
WARNING: errorprone disabled (skipped on builds not running inside CI environments, pass -Pvalidation.errorprone=true to enable)

> Task :solr:solrj:compileJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:solrj-streaming:compileJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:solrj-zookeeper:compileJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:core:compileJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:test-framework:compileJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:benchmark:compileJava
> Task :solr:benchmark:processResources
> Task :solr:benchmark:classes
> Task :solr:benchmark:compileTestJava
> Task :solr:benchmark:processTestResources
> Task :solr:benchmark:testClasses
> Task :gitStatus
> Task :solr:core:processResources
> Task :solr:core:classes
> Task :solr:core:jar
> Task :solr:solrj:processResources NO-SOURCE
> Task :solr:solrj:classes
> Task :solr:solrj:jar
> Task :solr:solrj-streaming:processResources
> Task :solr:solrj-streaming:classes
> Task :solr:solrj-streaming:jar
> Task :solr:solrj-zookeeper:processResources NO-SOURCE
> Task :solr:solrj-zookeeper:classes
> Task :solr:solrj-zookeeper:jar
> Task :solr:test-framework:processResources
> Task :solr:test-framework:classes
> Task :solr:test-framework:jar

> Task :solr:benchmark:test
:solr:benchmark:test (SUCCESS): 11 test(s)

> Task :solr:benchmark:wipeTaskTemp

> Task :solr:core:compileTestJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:core:processTestResources
> Task :solr:core:testClasses

> Task :solr:modules:analysis-extras:compileJava
Note: /solr/solr/modules/analysis-extras/src/java/org/apache/solr/schema/ICUCollationField.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:analysis-extras:processResources NO-SOURCE
> Task :solr:modules:analysis-extras:classes
> Task :solr:modules:analysis-extras:jar
> Task :solr:core:test
WARNING: Test org.apache.solr.TestRandomFaceting wrote 11,985,618 bytes of output.
WARNING: Test org.apache.solr.schema.TestCloudSchemaless wrote 12,347,875 bytes of output.
:solr:core:test (SUCCESS): 4651 test(s), 216 skipped

> Task :solr:core:wipeTaskTemp

> Task :solr:prometheus-exporter:compileJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:prometheus-exporter:processResources NO-SOURCE
> Task :solr:prometheus-exporter:classes

> Task :solr:prometheus-exporter:compileTestJava
Note: /solr/solr/prometheus-exporter/src/test/org/apache/solr/prometheus/exporter/MetricsQueryTemplateTest.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:prometheus-exporter:processTestResources
> Task :solr:prometheus-exporter:testClasses

> Task :solr:prometheus-exporter:test
:solr:prometheus-exporter:test (SUCCESS): 21 test(s)

> Task :solr:prometheus-exporter:wipeTaskTemp
> Task :solr:server:compileJava SKIPPED
> Task :solr:server:processResources NO-SOURCE
> Task :solr:server:classes UP-TO-DATE
> Task :solr:server:compileTestJava NO-SOURCE
> Task :solr:server:processTestResources NO-SOURCE
> Task :solr:server:testClasses UP-TO-DATE
> Task :solr:server:test NO-SOURCE
> Task :solr:server:wipeTaskTemp
> Task :solr:solr-ref-guide:compileJava NO-SOURCE
> Task :solr:solr-ref-guide:processResources NO-SOURCE
> Task :solr:solr-ref-guide:classes UP-TO-DATE

> Task :solr:solr-ref-guide:compileTestJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:solr-ref-guide:copySolrjTestResources
> Task :solr:solr-ref-guide:processTestResources NO-SOURCE
> Task :solr:solr-ref-guide:testClasses

> Task :solr:solr-ref-guide:test
:solr:solr-ref-guide:test (SUCCESS): 32 test(s)

> Task :solr:solr-ref-guide:wipeTaskTemp

> Task :solr:solrj:compileTestJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:solrj:processTestResources
> Task :solr:solrj:testClasses
> Task :solr:solrj:test
:solr:solrj:test (SUCCESS): 1087 test(s), 11 skipped

> Task :solr:solrj:wipeTaskTemp

> Task :solr:solrj-streaming:compileTestJava
Note: /solr/solr/solrj-streaming/src/test/org/apache/solr/client/solrj/io/stream/StreamingTest.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:solrj-streaming:processTestResources
> Task :solr:solrj-streaming:testClasses

> Task :solr:modules:sql:compileJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:sql:processResources
> Task :solr:modules:sql:classes
> Task :solr:modules:sql:jar
> Task :solr:solrj-streaming:test
:solr:solrj-streaming:test (SUCCESS): 604 test(s), 7 skipped

> Task :solr:solrj-streaming:wipeTaskTemp
> Task :solr:solrj-zookeeper:compileTestJava
> Task :solr:solrj-zookeeper:processTestResources
> Task :solr:solrj-zookeeper:testClasses

> Task :solr:solrj-zookeeper:test
:solr:solrj-zookeeper:test (SUCCESS): 15 test(s)

> Task :solr:solrj-zookeeper:wipeTaskTemp

> Task :solr:test-framework:compileTestJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:test-framework:processTestResources
> Task :solr:test-framework:testClasses

> Task :solr:test-framework:test
:solr:test-framework:test (SUCCESS): 17 test(s)

> Task :solr:test-framework:wipeTaskTemp
> Task :solr:webapp:compileJava NO-SOURCE
> Task :solr:webapp:processResources NO-SOURCE
> Task :solr:webapp:classes UP-TO-DATE
> Task :solr:webapp:compileTestJava NO-SOURCE
> Task :solr:webapp:processTestResources NO-SOURCE
> Task :solr:webapp:testClasses UP-TO-DATE
> Task :solr:webapp:test NO-SOURCE
> Task :solr:webapp:wipeTaskTemp

> Task :solr:modules:analysis-extras:compileTestJava
Note: /solr/solr/modules/analysis-extras/src/test/org/apache/solr/schema/TestICUCollationFieldUDVAS.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:analysis-extras:processTestResources
> Task :solr:modules:analysis-extras:testClasses

> Task :solr:modules:analysis-extras:test
:solr:modules:analysis-extras:test (SUCCESS): 31 test(s)

> Task :solr:modules:analysis-extras:wipeTaskTemp

> Task :solr:modules:analytics:compileJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:analytics:processResources NO-SOURCE
> Task :solr:modules:analytics:classes

> Task :solr:modules:analytics:compileTestJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:analytics:processTestResources
> Task :solr:modules:analytics:testClasses
> Task :solr:modules:analytics:test
:solr:modules:analytics:test (SUCCESS): 472 test(s)

> Task :solr:modules:analytics:wipeTaskTemp
> Task :solr:modules:clustering:compileJava
> Task :solr:modules:clustering:processResources NO-SOURCE
> Task :solr:modules:clustering:classes
> Task :solr:modules:clustering:compileTestJava
> Task :solr:modules:clustering:processTestResources
> Task :solr:modules:clustering:testClasses

> Task :solr:modules:clustering:test
:solr:modules:clustering:test (SUCCESS): 15 test(s)

> Task :solr:modules:clustering:wipeTaskTemp

> Task :solr:modules:extraction:compileJava
Note: /solr/solr/modules/extraction/src/java/org/apache/solr/handler/extraction/ExtractingDocumentLoader.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:extraction:processResources
> Task :solr:modules:extraction:classes

> Task :solr:modules:extraction:compileTestJava
Note: /solr/solr/modules/extraction/src/test/org/apache/solr/handler/extraction/ExtractingRequestHandlerTest.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:extraction:processTestResources
> Task :solr:modules:extraction:testClasses

> Task :solr:modules:extraction:test
:solr:modules:extraction:test (SUCCESS): 20 test(s)

> Task :solr:modules:extraction:wipeTaskTemp

> Task :solr:modules:gcs-repository:compileJava
Note: /solr/solr/modules/gcs-repository/src/java/org/apache/solr/gcs/GCSConfigParser.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:gcs-repository:processResources NO-SOURCE
> Task :solr:modules:gcs-repository:classes

> Task :solr:modules:gcs-repository:compileTestJava
Note: /solr/solr/modules/gcs-repository/src/test/org/apache/solr/gcs/ConcurrentDelegatingStorage.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:gcs-repository:processTestResources
> Task :solr:modules:gcs-repository:testClasses

> Task :solr:modules:gcs-repository:test
:solr:modules:gcs-repository:test (SUCCESS): 14 test(s), 1 skipped

> Task :solr:modules:gcs-repository:wipeTaskTemp

> Task :solr:modules:hadoop-auth:compileJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:hadoop-auth:processResources NO-SOURCE
> Task :solr:modules:hadoop-auth:classes

> Task :solr:modules:hadoop-auth:compileTestJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:hadoop-auth:copySolrCoreTestResources
> Task :solr:modules:hadoop-auth:processTestResources
> Task :solr:modules:hadoop-auth:testClasses
> Task :solr:modules:hadoop-auth:test
:solr:modules:hadoop-auth:test (SUCCESS): 55 test(s), 1 skipped

> Task :solr:modules:hadoop-auth:wipeTaskTemp

> Task :solr:modules:hdfs:compileJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:hdfs:processResources NO-SOURCE
> Task :solr:modules:hdfs:classes

> Task :solr:modules:hdfs:compileTestJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:hdfs:copySolrCoreTestResources
> Task :solr:modules:hdfs:processTestResources
> Task :solr:modules:hdfs:testClasses
> Task :solr:modules:hdfs:test
:solr:modules:hdfs:test (SUCCESS): 101 test(s), 41 skipped

> Task :solr:modules:hdfs:wipeTaskTemp
> Task :solr:modules:jaegertracer-configurator:compileJava
> Task :solr:modules:jaegertracer-configurator:processResources NO-SOURCE
> Task :solr:modules:jaegertracer-configurator:classes
> Task :solr:modules:jaegertracer-configurator:compileTestJava
> Task :solr:modules:jaegertracer-configurator:processTestResources
> Task :solr:modules:jaegertracer-configurator:testClasses

> Task :solr:modules:jaegertracer-configurator:test
:solr:modules:jaegertracer-configurator:test (SUCCESS): 1 test(s)

> Task :solr:modules:jaegertracer-configurator:wipeTaskTemp

> Task :solr:modules:jwt-auth:compileJava
Note: /solr/solr/modules/jwt-auth/src/java/org/apache/solr/security/jwt/JWTAuthPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:jwt-auth:processResources NO-SOURCE
> Task :solr:modules:jwt-auth:classes

> Task :solr:modules:jwt-auth:compileTestJava
Note: /solr/solr/modules/jwt-auth/src/test/org/apache/solr/security/jwt/JWTAuthPluginIntegrationTest.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:jwt-auth:processTestResources
> Task :solr:modules:jwt-auth:testClasses
> Task :solr:modules:jwt-auth:test
:solr:modules:jwt-auth:test (SUCCESS): 61 test(s)

> Task :solr:modules:jwt-auth:wipeTaskTemp

> Task :solr:modules:langid:compileJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:langid:processResources
> Task :solr:modules:langid:classes
> Task :solr:modules:langid:compileTestJava
> Task :solr:modules:langid:processTestResources
> Task :solr:modules:langid:testClasses

> Task :solr:modules:langid:test
:solr:modules:langid:test (SUCCESS): 43 test(s)

> Task :solr:modules:langid:wipeTaskTemp
> Task :solr:modules:ltr:compileJava
> Task :solr:modules:ltr:processResources NO-SOURCE
> Task :solr:modules:ltr:classes

> Task :solr:modules:ltr:compileTestJava
Note: /solr/solr/modules/ltr/src/test/org/apache/solr/ltr/TestLTROnSolrCloud.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:ltr:processTestResources
> Task :solr:modules:ltr:testClasses
> Task :solr:modules:ltr:test
:solr:modules:ltr:test (SUCCESS): 169 test(s)

> Task :solr:modules:ltr:wipeTaskTemp
> Task :solr:modules:opentelemetry:compileJava
> Task :solr:modules:opentelemetry:processResources NO-SOURCE
> Task :solr:modules:opentelemetry:classes
> Task :solr:modules:opentelemetry:compileTestJava
> Task :solr:modules:opentelemetry:processTestResources
> Task :solr:modules:opentelemetry:testClasses

> Task :solr:modules:opentelemetry:test
:solr:modules:opentelemetry:test (SUCCESS): 5 test(s)

> Task :solr:modules:opentelemetry:wipeTaskTemp
> Task :solr:modules:s3-repository:compileJava
> Task :solr:modules:s3-repository:processResources NO-SOURCE
> Task :solr:modules:s3-repository:classes
> Task :solr:modules:s3-repository:compileTestJava
> Task :solr:modules:s3-repository:processTestResources
> Task :solr:modules:s3-repository:testClasses
> Task :solr:modules:s3-repository:test
:solr:modules:s3-repository:test (SUCCESS): 42 test(s), 1 skipped

> Task :solr:modules:s3-repository:wipeTaskTemp

> Task :solr:modules:scripting:compileJava
Note: /solr/solr/modules/scripting/src/java/org/apache/solr/scripting/xslt/XSLTUpdateRequestHandler.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:scripting:processResources NO-SOURCE
> Task :solr:modules:scripting:classes

> Task :solr:modules:scripting:compileTestJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :solr:modules:scripting:processTestResources
> Task :solr:modules:scripting:testClasses

> Task :solr:modules:scripting:test
:solr:modules:scripting:test (SUCCESS): 22 test(s), 1 skipped

> Task :solr:modules:scripting:wipeTaskTemp
> Task :solr:modules:sql:compileTestJava
> Task :solr:modules:sql:copySolrCoreTestResources
> Task :solr:modules:sql:processTestResources
> Task :solr:modules:sql:testClasses

> Task :solr:modules:sql:test
:solr:modules:sql:test (SUCCESS): 33 test(s)

> Task :solr:modules:sql:wipeTaskTemp
The slowest tests (exceeding 500 ms) during this run:
  66.44s TestSolrJErrorHandling.testWithXml (:solr:solrj)
  53.54s PeerSyncReplicationTest.test (:solr:core)
  50.47s ReplicationFactorTest.test (:solr:core)
  47.42s SyncSliceTest.test (:solr:core)
  40.84s BasicDistributedZkTest.test (:solr:core)
  36.75s ReindexCollectionTest.testSameTargetReindexing (:solr:core)
  36.06s TestDistributedStatsComponentCardinality.test (:solr:core)
  35.57s UnloadDistributedZkTest.test (:solr:core)
  34.74s TestCloudSearcherWarming.testPeersyncFailureReplicationSuccess (:solr:core)
  34.11s TestCloudSchemaless.test (:solr:core)
The slowest suites (exceeding 1s) during this run:
  118.23s TestSolrJErrorHandling (:solr:solrj)
  83.39s TestRecovery (:solr:core)
  78.38s MultiThreadedOCPTest (:solr:core)
  72.02s ReindexCollectionTest (:solr:core)
  63.47s SchemaTest (:solr:solrj)
  60.36s TestTlogReplica (:solr:core)
  54.07s PeerSyncReplicationTest (:solr:core)
  51.01s ReplicationFactorTest (:solr:core)
  49.87s TestPackages (:solr:core)
  49.46s TestCloudConsistency (:solr:core)

BUILD SUCCESSFUL in 2h 2m 35s
138 actionable tasks: 136 executed, 2 up-to-date
------------------solr:install_&_test_both_success-------------------------
https://github.com/apache/solr solr
solr  |  https://github.com/apache/solr | releases/solr/9.2.0 | GitHub  | Pass |  Both_Install_and_Test_Success

**Comment by shubham-garud** [comment]
Have disabled Travis check as the execution time is more than 50 minutes.
Already attached logs above.

File: a/apache-solr/solr_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+export PATH=/usr/lib/jvm/java-11-openjdk-11.0.12.0.7-0.el8_4.ppc64le/bin/:$PATH
+
+
+git clone $PACKAGE_URL
+cd  $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+chmod +x gradlew
+
+if ! ./gradlew localSettings; then
```
Comment (nikhil-kalbande) [inline a/apache-solr/solr_ubi_8.5.sh:40]: @shubham-garud Is this command mentioned readme or CI workflow? Can you please point the reference

Comment (shubham-garud) [reply a/apache-solr/solr_ubi_8.5.sh:40]: Yes @nikhil-kalbande 
Here is the reference :
https://github.com/apache/solr/actions/runs/4028015923/workflow

File: a/apache-solr/build_info.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "package_name":"solr",
```
Comment (nikhil-kalbande) [inline a/apache-solr/build_info.json:2]: @shubham-garud Here also make it `apache-solr`

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**